src/Identity/Extensions.Core/src/LockoutOptions.cs

```diff
@@ -32,4 +32,10 @@ public class LockoutOptions
     /// </summary>
     /// <value>The <see cref="TimeSpan"/> a user is locked out for when a lockout occurs.</value>
     public TimeSpan DefaultLockoutTimeSpan { get; set; } = TimeSpan.FromMinutes(5);
+
+    /// <summary>
+    /// Specifies whether the lockout should be permanent.
+    /// If true, the user will be locked out indefinitely.
```

Comment:
Feel free to reject. The MSFT style guide avoids future tense unless it's needed or harder to understand.  What does indefinitely mean and how does it differ from "is locked out". Locked out until it's set to `false`, but that's not indefinitely
```suggestion
    /// If true, the user is locked out.
```